CHANGELOG.md

```diff
@@ -207,6 +208,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - the craft objective could be completed without consuming materials
 - LocationObjective resolves variable for a player who does not have the objective, and so maybe also dont have the variable
 - performance when using many conditions / events / ... repeatedly
+- AsyncSaver runs dead on connection loss, resulting in inability to reconnect to database
```

Comment:
I don't thing this formulation is good, but I'm also not entirely happy with what I came up:
```suggestion
- AsyncSaver shut itself down if the database connection died, causing data loss
```